gradle/testDependencies.gradle

```diff
@@ -19,7 +19,7 @@ ext {
 
 // Test classpath libraries
 testLibraries.spock =   'org.spockframework:spock-core:1.0-groovy-2.4'
-testLibraries.jsoup =   'org.jsoup:jsoup:1.6.3'
+testLibraries.jsoup =   'org.jsoup:jsoup:1.11.2'
```

Comment:
This dependency change is on test dependencies only, not for the build logic. If updated, it should be updated in `buildSrc/build.gradle.kts` too. I appended a commit to remove the upgrade as it is not required by this set of changes and would impact documentation generation thus is better handled as a separate PR.